src/com/codefortomorrow/beginner/chapter2/solutions/VariableTypes.java

```diff
@@ -0,0 +1,34 @@
+package com.codefortomorrow.beginner.chapter2.solutions;
+
+/**
+ * @author ArmeetJatyani
+ * March 2021
+ * 
+ * Define different types of variables
+ */
+
+ public class VariableTypes {
+    public static void main(String[] args) {
+        // write your code here
+        
+        // define an integer variable on line 15
+        int age = 23;
+        
+        // define a float variable on line 18
+        float pi = 3.14f;
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [CNT: Rough value of known constant found (CNT_ROUGH_CONSTANT_VALUE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#cnt-rough-value-of-known-constant-found-cnt-rough-constant-value).
It's recommended to use the predefined library constant for code clarity and better precision.

Reply:
fixed